css/style.css

```diff
@@ -0,0 +1,2043 @@
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.hd-color {
+  color: #f42;
+  font-size: 27px;
+  font-weight: 800;
+}
+
+.hd-color-only {
+  color: #f42;
+}
+
+.hd-color:hover {
+  color: black;
+}
+
+.nav-mobile-2 p {
+  padding: 0 1rem;
+  box-sizing: border-box;
+}
+
+.sub-section-hd p {
+  margin-left: 10px;
+}
+
+.latest-funding p {
+  font-size: 30px;
+  margin-left: 10px;
+}
+
+.bt-footer p {
+  margin: 0 30px;
+  color: #aaa;
+}
+
+.fl-1 div p {
+  line-height: 1.5;
+}
+
+.containers-latest-deals p:hover {
+  cursor: pointer;
+  color: #f42;
+}
+
+header {
+  top: 0;
+  position: fixed;
+  opacity: 1;
+  width: 100%;
+  z-index: 10;
+  height: 155px;
+}
+
+.img-pad {
+  display: none;
+}
+
+main {
+  height: auto;
+  max-width: 100%;
+  position: relative;
+  margin-top: 100px;
+}
+
+.ipad-margin-lft {
+  opacity: 1;
+}
+
+.ipad-nav-img {
+  display: none;
+}
+
+.nav-mobile {
+  height: 55px;
+  width: 100%;
+  opacity: 1;
+  background-color: white;
+}
+
+.nav-ipad-2 {
+  display: none;
+  opacity: 1;
+}
+
+.nav-ipad {
+  display: none;
+}
+
+.mb-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  width: 100%;
+  position: relative;
+}
+
+.mb-container img {
+  height: 50px;
+  width: 5rem;
+  padding: 0 1.5rem;
+  background-color: red;
+  fill: #fff;
+  box-sizing: border-box;
+}
+
+.mb-container ul {
+  height: 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  position: absolute;
+  top: 0;
+  right: 0;
+}
+
+.fl-1 a {
+  color: black;
+}
+
+.latest-funding p a {
+  color: red;
+}
+
+.mb-container ul li a {
+  margin: 0 1rem;
+  border-left: 1px solid #d3d3d3;
+  padding: 1rem;
+  box-sizing: border-box;
+}
+
+.fa-search {
+  color: #333;
+}
+
+.fa-bars {
+  color: red;
+}
+
+.nav-mobile-2 {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #fd3;
+  height: 50px;
+  margin-top: -4.7px;
+  line-height: 1.3;
+  width: 100%;
+  opacity: 1;
+}
+
+.grid-sec {
+  height: auto;
+  width: 100%;
+  position: relative;
+}
+
+.main-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr 1fr;
+  position: relative;
+  height: 600px;
+  width: 100%;
+}
+
+.main-grid .box1 {
+  grid-row: 1/2;
+  grid-column: 1/2;
+  background: rgba(0, 0, 0, 0.25);
+  position: relative;
+  color: white;
+  height: 200px;
+  width: 100%;
+}
+
+.containers-latest-deals div {
+  margin-top: 10px;
+  line-height: 1.3;
+}
+
+.main-grid .box1 div {
+  position: absolute;
+  top: 0;
+  width: 95%;
+  height: 200px;
+  font-size: 1.2rem;
+  line-height: 1.5;
+  padding-left: 1.5%;
+  box-sizing: border-box;
+  background-color: red 5px solid;
```

Comment:
This is an invalid property. `background-color` only takes color values. If you want to add other background properties, use the shorthand `background` property.